app/controllers/storage/FlashExtractor.scala

```diff
@@ -27,4 +27,11 @@ class FlashExtractor {
   def extractMovementType(request: Request[_]): Option[JourneyType] =
     extractValue(FlashKeys.MOVEMENT_TYPE, request).map(JourneyType.withName)
 
+  def extractUcr(request: Request[_]): Option[String] =
+    extractValue(FlashKeys.UCR, request)
+}
+
+object FlashKeys {
```

Comment:
I know, I'm a bit pedantic... still `object FlashKeys`? Not `object FlashExtractor`?

Reply:
My mistake, corrected now.